.github/workflows/sycl-rel-nightly.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       # We upload the build for people to download/use, override its name and
       # prefer widespread gzip compression.
       artifact_archive_name: sycl_linux.tar.gz
+      pack_release: 'true'
```

Comment:
should this be in the other pr?

Reply:
I don't know, would it be better to not mix these changes?:)
Anyway, now we may get a "release" build for both OS, so I either need to update this PR or create a new one If you wish.

Reply:
nah its fine i wasnt sure if the other one would be merged so quickly